Dockerfile

```diff
@@ -1,14 +1,8 @@
 FROM google/cloud-sdk:170.0.1-slim
 
-RUN pip install virtualenv
-RUN virtualenv -q ~/.firecloud-tools/venv
-RUN /bin/bash -c "source ~/.firecloud-tools/venv/bin/activate"
+COPY install.sh /
+RUN /bin/bash -c "/install.sh"
```

Comment:
`RUN /install.sh` should be all that's needed. Anything inside the RUN command is run as `/bin/sh -c <run command>` https://docs.docker.com/engine/reference/builder/#run